tests/examples/conftest.py

```diff
@@ -27,9 +29,22 @@
 File = collections.namedtuple('File', 'path content')
 
 
-def pytest_generate_tests(metafunc):
+def get_tftest_directive(s: str) -> typing.Optional[str]:
```

Comment:
Please don't use typed Python in this repo.

Reply:
removed.